var/spack/repos/builtin/packages/py-onnx-runtime/package.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyOnnxRuntime(CMakePackage):
+    """ONNX Runtime is a performance-focused complete scoring
+    engine for Open Neural Network Exchange (ONNX) models, with
+    an open extensible architecture to continually address the
+    latest developments in AI and Deep Learning. ONNX Runtime
+    stays up to date with the ONNX standard with complete
+    implementation of all ONNX operators, and supports all
+    ONNX releases (1.2+) with both future and backwards
+    compatibility."""
+
+    homepage = "https://github.com/microsoft/onnxruntime"
+    git      = "https://github.com/microsoft/onnxruntime.git"
+
+    version('1.7.2',  tag='v1.7.2',  submodules=True)
+
+    variant('cuda', default=False, description='Build with CUDA support')
+
+    depends_on('cmake@3.1:', type='build')
+    depends_on('ninja', type='build')
+    depends_on('python', type='build')
+    depends_on('protobuf')
```

Comment:
I think there should also be a dep on py-protobuf

Reply:
Maybe. It will come from py-onnx anyway.

Reply:
If the package has a direct dependency on `py-protobuf`, you should add one. In #27798, I found a bug where we were adding build deps of build deps to the `PYTHONPATH` and fixed it.

Reply:
Done